tests/dags/test_on_kill.py

```diff
@@ -25,13 +25,12 @@
 
 class DummyWithOnKill(DummyOperator):
     def execute(self, context):
-        time.sleep(10)
+        time.sleep(1)
```

Comment:
This full sleep should never be hit, but leaving it at 10s seems safer.

Reply:
I have never experienced a problem with it but I will leave 10s.

Reply:
It would _probably_ be find at 1s, except if it happens to run somewhere under crazy load :)